src/reactor.cc

```diff
@@ -21,15 +21,43 @@ Reactor::Reactor(cyclus::Context* ctx)
       refuel_time(0),
       cycle_step(0),
       power_cap(0),
+      power_commod("power"),
       discharged(false) {
   cyclus::Warn<cyclus::EXPERIMENTAL_WARNING>(
       "the Reactor archetype "
       "is experimental");
 }
 
+#pragma cyclus def clone cycamore::Reactor
+
+#pragma cyclus def schema cycamore::Reactor
+
+#pragma cyclus def annotations cycamore::Reactor
+
+#pragma cyclus def infiletodb cycamore::Reactor
+
+#pragma cyclus def snapshot cycamore::Reactor
+
+#pragma cyclus def snapshotinv cycamore::Reactor
+
+#pragma cyclus def initinv cycamore::Reactor
+
+void Reactor::InitFrom(Reactor* m) {
+  #pragma cyclus impl initfromcopy cycamore::Reactor
+  cyclus::toolkit::CommodityProducer::Copy(m);
+}
+
+void Reactor::InitFrom(cyclus::QueryableBackend* b) {
+  #pragma cyclus impl initfromdb cycamore::Reactor
+
+  namespace tk = cyclus::toolkit;
+  tk::CommodityProducer::Add(tk::Commodity(power_commod),
+                             tk::CommodInfo(power_cap, power_cap));
```

Comment:
not needed for now. we can update if needed later (it would be optional in any case).